src/index.js

```diff
@@ -1,46 +1,72 @@
 import './style.css';
+import storage from './localStorage.js';
+import completed from './markCompletion.js';
+import populateUI from './populateUI.js';
 
-const items = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Complete To Do List project',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Watch Documentary ',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Family meeting',
-    completed: false,
-    index: 4,
-  },
-];
-
-const createList = () => {
-  const listContainer = document.querySelector('.todo-container');
-  const unorderedList = document.createElement('ul');
-  unorderedList.classList.add('listItems');
-  listContainer.appendChild(unorderedList);
-
-  items.forEach((task) => {
-    const list = document.createElement('li');
-    list.classList.add('list');
-    list.innerHTML = `
-    <div class='task-container'>
-      <div class='task'>
-        <input class="checkbox" type="checkbox"/>
-        <label id="${task.index}" class="label">${task.description}</label> 
-      </div>  
-      <i class="fa-solid fa-ellipsis-vertical fa-icon"></i>
-    </div>`;
-    unorderedList.appendChild(list);
+const todoList = [];
+
+const populateScreen = () => {
+  if (!JSON.parse(localStorage.getItem('todo-list'))) {
+    storage.storetask(storage.sortindex(todoList));
+  }
+  // Display List of tasks in UI
+  populateUI.displayList();
+
+  // Set up checkbox event
+  const checkboxes = document.querySelectorAll('.checkbox');
+
+  [...checkboxes].forEach((button) => {
+    button.addEventListener('change', completed);
+  });
+
+  // Event for Update Task
+  const inputs = document.querySelectorAll('.description');
+  [...inputs].forEach((input) => {
+    input.addEventListener('focusout', storage.updateTask);
+  });
+
+  // Icon and Trash can toggle
+  [...inputs].forEach((input) => {
+    input.addEventListener('focus', (event) => {
+      event.target.style.backgroundColor = '#fff4bf';
+      event.target.parentElement.style.backgroundColor = '#fff4bf';
+      event.target.nextSibling.classList.add('hideellipsis');
+      event.target.nextSibling.nextSibling.classList.remove('hideellipsis');
+    });
+  });
+
+  // Trash task from list
+  const trashes = document.querySelectorAll('.fa-trash');
+  [...trashes].forEach((trash) => {
+    trash.addEventListener('click', (event) => {
+      storage.remove(event);
+      const oldList = document.querySelectorAll('.todoItem');
+      [...oldList].forEach((e) => e.remove());
+      populateScreen();
+    });
```

Comment:
- [ ] When I click on the trash icon to delete any task nothing happens (see the GIF below). So please fix that.

![chrome-capture-2022-6-8](https://user-images.githubusercontent.com/68796232/178010068-616d08b7-8952-4e23-bffd-78635dce34a8.gif)